sslcontext-kickstart/src/main/java/nl/altindag/ssl/SSLFactory.java

```diff
@@ -396,6 +412,31 @@ public Builder withTrustMaterial(Set<X509Certificate> certificates, TrustAnchorT
             return withTrustMaterial(trustStore, trustOptions);
         }
 
+        public Builder withPlaceHolderIdentityMaterial() {
+            return withPlaceHolderMaterial(
+                    "javax.net.ssl.keyStore",
+                    "javax.net.ssl.keyStorePassword",
+                    "javax.net.ssl.keyStoreType",
+                    this::withIdentityMaterial
+            );
+        }
+
+        private Builder withPlaceHolderMaterial(String keyStorePathSystemProperty, String keyStorePasswordProperty, String keyStoreTypeProperty, TriConsumer<Path, char[], String> keyStorePropertyConsumer) {
+            Path keystore = Optional.ofNullable(System.getProperty(keyStorePathSystemProperty))
```

Comment:
`keyStore = ""` (empty string) should be treated the same as there is no system property provided.

Reply:
Good suggestion, fixed it